src/components/App.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+
+import Counter from './counter';
+
+function App() {
+  const [state, setState] = useState({
+    count: 0,
+  });
+  const { count } = state;
+  const buttons = [
+    { id: 1, value: 1 },
+    { id: 2, value: 2 },
+    { id: 3, value: 3 },
+    { id: 4, value: 4 },
+    { id: 5, value: 5 },
+  ];
+
+  function handleClick(addedNumber = 1) {
+    setState({
+      count: count + addedNumber,
+    });
+  }
+  return (
+    <div>
+      <Counter
+        count={count}
+        buttons={buttons}
+        onClick={handleClick}
+      />
+    </div>
+  );
+}
```

Comment:
```suggestion
  const [state, setState] = useState({
    count: 0,
  });
  
  const { count } = state;
  
  const buttons = [
    { id: 1, value: 1 },
    { id: 2, value: 2 },
    { id: 3, value: 3 },
    { id: 4, value: 4 },
    { id: 5, value: 5 },
  ];

  function handleClick(addedNumber = 1) {
    setState({
      count: count + addedNumber,
    });
  }
  
  return (
    <div>
      <Counter
        count={count}
        buttons={buttons}
        onClick={handleClick}
      />
    </div>
  );
}
```

우리가 글을 작성할 때 내용에 따라 문단을 나눠 주듯이 코드에도 마찬가지에요.

의미상으로 구분되어지는 코드는 공백을 넣어주면 코드 가독성이 더 좋아져요.

Reply:
네 훨씬 눈에 잘들어오네요